rem/Search.swift

```diff
@@ -96,22 +96,31 @@ struct FilterPicker: View {
     var body: some View {
         VStack(alignment: .leading) {
             Picker("Application", selection: $selectedFilterAppIndex) {
-                ForEach(applicationNameFilter.indices, id: \.self) { index in
-                    Text(applicationNameFilter[index])
+                ForEach(applicationFilterArray.indices, id: \.self) { index in
+                    Text(applicationFilterArray[index])
                         .tag(index)
                 }
             }
+            .onHover(perform: { hovering in
+              updateAppFilterData()
+            })
```

Comment:
why `onHover` out of curiosity? This means it's blank until you hover. Could we instead do `onAppear`?

Reply:
I had initially implemented it with `onAppear`, but it would cause the filter to only update on the first time the user clicked on "Search". It means that, if you left the search screen and went back to it again, the filter would stay the same until you restarted the program.

I made a new change adding both `onAppear` and `onHover`, which solves the issue of it being blank at first and not updating after the first search.